src/app/component/dashboard/marketing/massive-email/massive-email.component.scss

```diff
@@ -0,0 +1,51 @@
+.button-padding > button {
```

Comment:
This looks like the same code, but repeated two times(for email and SMS). Why?
Why we don't make this one global SCSS file and use one code in different places...